controllers/spiaccesstokenbinding_controller.go

```diff
@@ -113,16 +162,35 @@ func (r *SPIAccessTokenBindingReconciler) SetupWithManager(mgr ctrl.Manager) err
 
 			})
 			if err != nil {
-				spiAccessTokenBindingLog.Error(err, "failed to list SPIAccessTokenBindings while determining the ones linked to Secret",
+				enqueueLog.Error(err, "failed to list SPIAccessTokenBindings while determining the ones linked to Secret",
 					"SecretName", o.GetName(), "SecretNamespace", o.GetNamespace())
 				return []reconcile.Request{}
 			}
+
+			logReconciliationRequests(requests, "SPIAccessTokenBinding", o, "Secret")
+
 			return requests
 		})).
+		Watches(&source.Kind{Type: &corev1.ServiceAccount{}}, handler.EnqueueRequestsFromMapFunc(func(o client.Object) []reconcile.Request {
```

Comment:
watch `ServiceAccounts` is another heavy object in kubernetes deployments. We already watch secrets and most probably it's cause of memory spikes after startup. Have you seen any substantial increase of memory consumption or spike on startup comparing to main branch? (ideally tested on OpenShift as it is more heavyveight ootb)

Reply:
I've looked at the graphs and the initial spike peaks at 112MB on an otherwise empty OpenShift cluster, while the latest operator image from main peaked at 94MB.

There are 351 service accounts in 69 namespaces in that cluster.

I guess we will have to wait and see how this pans out.